src/HealthGPS/hierarchical_model_static.h

```diff
@@ -1,33 +1,105 @@
 #pragma once
 
-#include "hierarchical_model_types.h"
-#include <functional>
+#include "interfaces.h"
+#include "mapping.h"
 
 namespace hgps {
 
+/// @brief Regression coefficients data type
+struct Coefficient {
+    /// @brief The coefficient value
+    double value{};
+
+    /// @brief Associated p-value
+    double pvalue{};
+
+    /// @brief Associated t-value
+    double tvalue{};
+
+    /// @brief Associated standard error
+    double std_error{};
+};
```

Comment:
Agree. Missing data is should be fatal. I plan to do a big input data sanitisation PR in future, but didn't want to touch it just yet, lest something breaks.

Reply:
FYI `std::optional` can be handy for these sorts of situations, because it lets you explicitly check whether a variable has been assigned a value or not.